src/components/DatePicker/DateRangePicker.jsx

```diff
@@ -229,7 +250,10 @@ DateRangePicker.propTypes = {
   displayFormat: PropTypes.string,
   label: PropTypes.node,
   placeholder: PropTypes.node,
-  navigationByYear: PropTypes.bool
+  navigationByYear: PropTypes.bool,
+  labelYear: PropTypes.string,
+  labelStartDate: PropTypes.string,
+  labelEndDate: PropTypes.string
```

Comment:
Mayne we will not use this only to pass strings, I think `PropTypes.node` is a better option

Reply:
6693dd4c5757ae26b356ee7422623c258ee968e5